packages/sources/finnhub-test/src/endpoint/quote.ts

```diff
@@ -0,0 +1,116 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+
+const logger = makeLogger('Finnhub quote endpoint')
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
```

Comment:
I think Finnhub v2 has some aliases missing here. Could you update this to
```suggestion
    aliases: ['quote', 'asset', 'from'],
```

Reply:
Done